RL_UserCard

lluneva opened the pull request.

Commit:
temp

Commit:
REFERENCE: Usercard and Userlist
DESCRIPTION: Usercard and Userlist components fixed

lluneva commented:
With static data

rokija commented on src/App.js:
```diff
@@ -2,11 +2,12 @@ import React, { Component } from "react";
 class App extends Component {
   render() {
     return (
       <React.Fragment>
```
why is React.Fragment used here?

rokija commented on src/App.js:
```diff
@@ -2,11 +2,12 @@ import React, { Component } from "react";
 import "./App.css";
 import Header from "./components/Header/Header";
 
+
```
remove empty line

rokija commented on src/components/Usercard/Usercard.css:
```diff
@@ -0,0 +1,21 @@
+.UserCard {
+  align-items: left;
+  text-align: left;
```
why both align-items and text-align?

rokija commented on src/components/Usercard/Usercard.css:
```diff
@@ -0,0 +1,21 @@
+  text-align: left;
+  padding-top: 1%;
+  border-bottom: 1px solid rgb(230, 227, 227);
+}
```
add new line

rokija commented on src/components/Usercard/Usercard.css:
```diff
@@ -0,0 +1,21 @@
+}
+.Usercard__head {
+  display: flex;
+}
```
add new line

rokija commented on src/components/Usercard/Usercard.css:
```diff
@@ -0,0 +1,21 @@
+}
+.UserCard__username {
+  margin-right: 20px;
+  margin-bottom: 0;
```
remove this

rokija commented on src/components/Usercard/Usercard.css:
```diff
@@ -0,0 +1,21 @@
+  margin-right: 20px;
+  margin-bottom: 0;
+  font-size: 1rem;
+}
```
new line

rokija commented on src/components/Usercard/Usercard.css:
```diff
@@ -0,0 +1,21 @@
+}
+.UserCard__content {
+  font-size: 12px;
+}
```
new line

rokija commented on src/components/Usercard/Usercard.css:
```diff
@@ -0,0 +1,21 @@
+.UserCard__content {
+  font-size: 12px;
+}
+.UserCard__editBtn {
```
rename to UserCard__edit-button

rokija commented on src/components/Usercard/Usercard.js:
```diff
@@ -0,0 +1,33 @@
+    const { username, email, level, id, goToUserId } = this.props;
+
+    return (
+      <React.Fragment>
```
why is React.Fragment used here?

rokija commented on src/components/Usercard/Usercard.js:
```diff
@@ -0,0 +1,33 @@
+    return (
+      <React.Fragment>
+        <div className="UserCard">
+          <div className="">
```
remove this div if its not needed

rokija commented on src/components/Usercard/Usercard.js:
```diff
@@ -0,0 +1,33 @@
+              <h4 className="UserCard__username">{username}</h4>
+              <button
+                className="UserCard__editBtn btn btn-secondary"
+                onClick={goToUserId}
```
what is this function doing? is it not editUser?

rokija replied:
ah I got it, it goes to a separate page where the user is displayed?

rokija commented on src/components/Userlist/Userlist.js:
```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import Usercard from "../Usercard/Usercard";
+
+class Userlist extends Component {
```
rename to UserList

rokija commented on src/components/Userlist/Userlist.js:
```diff
@@ -0,0 +1,67 @@
+    return (
+      <div className="Userlist">
+        <h2> Users</h2>
+        {/* {this.props.users && this.props.users.map((user, index) */}
```
remove commented out code

rokija reviewed (changes requested):
please add human readable title and description to this pull request

Commit:
Merge branch 'develop' of https://github.com/rokija/quiz-game into RL_UserCard

rokija commented:
![screenshot 2019-02-19 at 17 16 31](https://user-images.githubusercontent.com/25583616/53025782-3dd41e00-346a-11e9-9059-b94529dca24d.png)

janisknets committed:
REFERENCE: File renaming
DESCRIPTION:
   Fixed user list file naming

janisknets committed:
REFERENCE: UserCard Naming
DESCRIPTION:
   Moved usercard folder to be userCard (same with the files within the folder from usercard to userCard

rokija commented:
![screenshot 2019-02-20 at 09 33 58](https://user-images.githubusercontent.com/25583616/53074149-b1ba0900-34f2-11e9-9227-bc8990d5ce8f.png)

if I click on the edit button

Commit:
REFERENCE: RL_UserCard
DESCRIPTION: Fixed error on non-existent history;
Added Routes in App.js.

Commit:
Merge branch 'develop' of https://github.com/rokija/quiz-game into RL_UserCard

Commit:
Merge branch 'develop' of https://github.com/rokija/quiz-game into RL_UserCard
